docs/players/Installation_iOS.md

```diff
@@ -24,17 +24,21 @@ To install the signed ipa on your device, you can use Xcode or Apple Configurato
 
 Note: if you don't need in-game videos, you can omit downloading/copying file VIDEO.VID from data folder - it will save your time and space. The same applies to the Mp3 directory.
 
-### Step 2.a: Installing data files with Finder or Windows explorer
+### Step 2.a: Installing data files with GOG offline installer
+
+If you buyed HoMM3 on [GOG](https://www.gog.com/de/game/heroes_of_might_and_magic_3_complete_edition) you can download the files directly with the browser and install it in launcher. You need to select the exe file first and afterwards the bin file. This process takes a few seconds. Be patient.
```

Comment:
```suggestion
If you bought HoMM3 on [GOG](https://www.gog.com/de/game/heroes_of_might_and_magic_3_complete_edition), you can download the files directly from the browser and install them in the launcher. Select the .exe file first, then the .bin file. This may take a few seconds. Please be patient.
```